### PR TITLE
rmarkdown_tutorial

### PR DESCRIPTION
**Pull request by Alexsfwu**

### CODE REVIEW

**Comment by bojum** [comment]
@skyetim @Yingyu-Cao, will line 121 in `rmarkdown_tutorial.Rmd` cause a problem?

**Comment by skyetim** [comment]
> @skyetim @Yingyu-Cao, will line 121 in `rmarkdown_tutorial.Rmd` cause a problem?

Yes it will. I recommend taking a screenshot and putting the screenshot instead.

File: rmarkdown_tutorial.Rmd [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+* **headers**
+```{}
+# Header1 {#anchor}
+## Header 2 {#css_id}
+### Header 3 {.css_class}
+#### Header 4
+##### Header 5
+###### Header 6 
+```
+# Header1 {#anchor}
```
Comment (skyetim) [inline rmarkdown_tutorial.Rmd]: I recommend inserting a photo here. There should be no single # header other than the title.

**Review by skyetim — CHANGES_REQUESTED** [review]

File: rmarkdown_tutorial.Rmd [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+
+### 2.2. Open file
+You can create a new file or open existed from the directory that you choose.
+
+### 2.3. output format
+The Rmarkdown can render any Rmd file into a format that Rmarkdown supports. For example, the code below renders OutputExample.Rmd to a Microsoft Word document. 
+```{r results='hide'}
+library(rmarkdown)
+render("OutputExample.Rmd", output_format = "word_document")
```
Comment (skyetim) [inline rmarkdown_tutorial.Rmd]: this file doesn't exist.

**Review by skyetim — CHANGES_REQUESTED** [review]

**Comment by skyetim** [comment]
🎉 Congrats! Your contribution has been merged!

**Pull request merged**